### DISCUSSION TITLE
Users/Roles documentation is lacking

### DISCUSSION THREAD

**Discussion by kenske** [tolgee/tolgee-platform#1263] [General]
I just started setting up self-hosted Tolgee and the documentation around users/roles/permissions seems to be lacking. I have several questions:

- What role does the user created by `create-initial-user` have?
- It seems that this initial user does not have many permissions...how do I set the role for this user?
- I see that if I enable `user-can-create-projects` and `user-can-create-organizations` then the initial user can create the projects, but then so can all other users. How do I enable these permissions for a single user?

**Comment by JanCizmar** [comment]
Hello,

yep. The documentation and the current implementation is not complete. Because we don't have implemented server administration yet. And there is currently no way how to set users permissions for the whole server. There are issues for this:

https://github.com/tolgee/tolgee-platform/issues/974
https://github.com/tolgee/tolgee-platform/issues/976
https://github.com/tolgee/tolgee-platform/issues/1031

**Reply by kenske** [reply]
Is there a way to allow a single user to have `create-projects` permissions? Even if it's a workaround that involves editing the DB directly.

**Reply by cyyynthia** [reply]
The default user created when starting the Tolgee server for the first time is considered Administrator. You can therefore use the [`tolgee.authentication.user-can-create-projects`](https://tolgee.io/docs/platform/self_hosting/configuration#user-can-create-projects) configuration field to prevent project creation except from the admin user.

There is a slight UI bug described in #976 where users still see the create button, but they will not be able to actually do so.

**Reply by kenske** [reply]
Whenever I try to create a project with the default user, I get a `Your permissions are not sufficient for this operation` error. I'm only able to add projects if I enable `tolgee.authentication.user-can-create-projects`.

**Reply by JanCizmar** [reply]
I am going to fix this today. So you will be always able to create projects/organizations with admin user.

**Comment by JanCizmar** [comment]
Hi all. 

So I explored the code and the problem is, the initial user had `role = USER` not `role = ADMIN`, so if you modify the in the DB, you would be able to create both project and organization with this user.

I will continue with resolving related issues in #976.
